Doc/using/cmdline.rst

```diff
@@ -994,3 +994,71 @@ Debug-mode variables
    Need Python configured with the :option:`--with-trace-refs` build option.
 
    .. versionadded:: 3.11
+
+.. _using-on-finding-modules:
+
+Finding modules
+---------------
+
+A module search path is initialized when Python starts. This module search path
+may be accessed at :data:`sys.path`.
+
+The first entry in the module search path is the directory containing the input
+script (or the the current directory when no file is specified).
+
+On Windows a default Python installation adds the :file:`/Lib` directory and
+the :file:`/Lib/site-packages` directory to the module search path. These
+locations are relative to the :file:`python.exe` location. Standard modules
+are usually found in the Lib directory and third-party modules in the
+Lib/site-packages directory.
+
+On Unix the lib and lib/site-packages directories are added by looking for
+:file:`/exec_prefix/lib/pythonversion` , :file:`/exec_prefix/lib/pythonversion/site-packages` ,
```

Comment:
I feel like we're missing some kind of substitution character here? Something like `/{exec_prefix}/lib/python{version}`